### PR TITLE
fix(FEC-10564): keep the poster for display for an audio-only entry

### PR DESCRIPTION
**Pull request by Yuvalke**
### Description of the Changes

Issue: audio stream has a black screen when it plays.
Solution: check if it's audio by sources.type and keep the poster.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/player.js [hunk 1]
```diff
@@ -1842,6 +1852,7 @@ export default class Player extends FakeEventTarget {
    * @private
    */
   _handleAutoPlay(): void {
+    this._posterManager.show();
```
Comment (dan-ziv) [inline src/player.js:1855]: It won't cause flickering in case of video autoplay?

File: src/player.js [hunk 2]
```diff
@@ -1091,6 +1091,16 @@ export default class Player extends FakeEventTarget {
     );
   }
 
+  /**
+   * Checking if the current playback is audio only.
+   * @function isAudio
+   * @returns {boolean} - Whether playback is audio.
+   * @private
+   */
+  isAudio(): boolean {
```
Comment (dan-ziv) [inline src/player.js:1100]: Add this API in kaltura player?

Comment (Yuvalke) [reply src/player.js:1100]: It should be added, yes

File: src/player.js (hunk 1 above)
Comment (Yuvalke) [reply src/player.js:1855]: I didn't see a flicker

Comment (OrenMe) [reply src/player.js:1855]: In any case, this is logically not safe - if autoplay is set we shoudn't show a poster.
Need to move to where we know we decided to not show it.

Comment (dan-ziv) [reply src/player.js:1855]: @Yuvalke try with throttling the browser

**Review by OrenMe — APPROVED** [review]

**Pull request merged**